mlflow/pyfunc/__init__.py

```diff
@@ -503,6 +506,7 @@ def predict(self, context, model_input, params=None):
 DATA = "data"
 ENV = "env"
 MODEL_CONFIG = "config"
+_MODEL_CODE_PATH = "model_code_path"
```

Comment:
Can we reuse this var?

Reply:
We should import these constants (`MODEL_CONFIG` and `MODEL_CODE_PATH`) from models/model.py file - refer to [this PR](https://github.com/mlflow/mlflow/pull/11916/files)

Reply:
Since this PR is merged, can we rebase?